modules/persistent-storage-csi-snapshots-restore.adoc

```diff
@@ -7,11 +7,13 @@
 
 After your `VolumeSnapshot` object is bound, you can use that object to provision a new volume that is pre-populated with data from the snapshot.
 
+When `volumemsnapshot.status.readyToUse` = `true`, the restore is successful and a new persistent volume claim (PVC) is created.
+
 The volume snapshot content object is used to restore the existing volume to a previous state.
 
 .Prerequisites
 * Logged in to a running {product-title} cluster.
-* A persistent volume claim (PVC) created using a Container Storage Interface (CSI) driver that supports volume snapshots.
+* A PVC created using a Container Storage Interface (CSI) driver that supports volume snapshots.
 * A storage class to provision the storage back end.
 
```

Comment:
@bobfuru I prefer to add one more prerequisite like snapshot is created successfully and ready to use.
cc @qinpingli

Reply:
added